install/ansible/istio/tasks/enable_mutating_webhooks.yml

```diff
@@ -0,0 +1,39 @@
+- name: Download missing webbook certificate scripts
```

Comment:
After #4466 , we no longer need to generate certs and patch CA bundle manually. It will be done by Istio CA and webhook service itself.

Reply:
Cool! What istio version has that feature enabled?

Reply:
It will be in 0.8

Reply:
Thank you!

Other than using the version number, is there any other I can know whether or not such functionality is present in the deployed Istio version?

Reply:
version number is probably the most reliable way to do that.

Reply:
Just to be clear, for versions newer than `0.8.0`, the only thing that will need to be done to enable automatic sidecar injection, will be to install the sidecar configmap and the sidecar bundle?

Thank you

Reply:
Yes.

Reply:
Nice, I'll update the PR soon

Reply:
PR Updated :)